RELEASE_PROCESS.md

```diff
@@ -0,0 +1,58 @@
+# GraphiQL Monorepo Release Process
+
+This document should guide you through doing a release of GraphiQL. Much of
+the release process is automated, so just follow this process through and
+everything should go smoothly!
+
+Note: we run all `lerna` commands with `npx lerna` to ensure that we're using
+the local copy, not a globally installed copy that might be out-of-sync with
+our local copy.
+
+## Step 0: see what would change
+
+1. `npx lerna changed --long --toposort`
+
+Ref: [lerna changed](https://github.com/lerna/lerna/tree/master/commands/changed)
+
+## Step 1: cut a version
+
+1. Run `npx lerna version` and choose which version to go to for each package
+2. `git push && git push --tags` to push tags to GitHub
+3. Update changelogs _(TODO: what command pulls in the conventional changelog?)_
```

Comment:
the conventional changelog is generated from the commit messages, when you run `lerna version`